doc/source/install/cmake.rst

```diff
@@ -0,0 +1,253 @@
+.. -*- rst -*-
+
```

Comment:
We can remove this because we use `.rst` as extension.
(We used to use `.txt`.)

Reply:
I see.

Reply:
Fixed.